kbs/src/api_server.rs

```diff
@@ -282,6 +282,8 @@ pub(crate) async fn api(
                     .map_err(|_| Error::TokenNotFound)?;
 
                 let claims = core.token_verifier.verify(token).await?;
+                let init_data = claims
+                    .pointer("/submods/cpu0/ear.veraison.annotated-evidence/init_data_claims");
```

Comment:
since it's just a subset of the token claims, any reason why we shouldn't make the full token available instead?

Reply:
Yeah good point. There's a little bit of a tradeoff here. Giving plugin authors the full plugin could make the interface more powerful but it makes things more complicated for plugin authors. I couldn't think of a use case for a plugin using the whole token so I thought this would be a little more convenient. wdyt?